crates/typst/src/math/attach.rs

```diff
@@ -83,12 +89,20 @@ impl LayoutMath for Packed<AttachElem> {
             };
         }
 
+        let t = layout!(t, sup_style_chain)?;
+        let b = layout!(b, sub_style_chain)?;
+        if let Some(stretch) = base.stretch() {
+            let relative_to_width =
+                measure!(t, width).max(measure!(b, width)).max(stretch.from);
```

Comment:
I think including `stretch.from` here makes interactions between `math.stretch` and `math.attach` a bit inconsistent. For example, a long attachment can cause the base to be stretched, but a shorter one doesn't cause the base to be compressed. If having `width: auto` with attachments means sizing the base relative to the attachments, then it should go both ways IMO.

```typst
#let base = math.stretch($=$, width: 250%)

$ // This gets shortened.
  stretch(base, width: #50%)
  // But this doesn't.
  stretch(base)^"a" $
```